homeassistant/components/moehlenhoff_alpha2/__init__.py

```diff
@@ -0,0 +1,124 @@
+"""Support for the Moehlenhoff Alpha2."""
+import asyncio
+import logging
+import time
+
+from moehlenhoff_alpha2 import Alpha2Base
+
+from homeassistant import exceptions
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+PLATFORMS = ["climate"]
+
+
+class CannotConnect(exceptions.HomeAssistantError):
+    """Error to indicate we cannot connect."""
+
+
+async def async_setup(hass: HomeAssistant, config: dict):
+    """Set up Alpha2 integration."""
+    return True
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
+    """Set up a config entry."""
+    base = Alpha2Base(entry.data["host"])
+    try:
+        await base.update_data()
+    except TimeoutError as err:
+        raise exceptions.ConfigEntryNotReady from err
+    base_uh = Alpha2BaseUpdateHandler(base, 60)
+    hass.data.setdefault(DOMAIN, {"connections": {}, "devices": set()})
```

Comment:
seems these are not linked to this entry_id, why not add this under the entry_id, so each entry keeps track of their own connections and devices

Reply:
Your right, that makes sense. Done.